backup

```diff
@@ -29,7 +29,7 @@ start=`date +%s`
 echo Starting Backup at $(date +"%Y-%m-%d %H:%M:%S")
 
 set +e
-restic backup ${RESTIC_BACKUP_ARGS:-} ${tag_options} ${RESTIC_BACKUP_SOURCES}
+restic --repo="${RESTIC_REPOSITORY}" backup ${RESTIC_BACKUP_ARGS:-} ${tag_options} ${RESTIC_BACKUP_SOURCES}
```

Comment:
Is this just to make it more obvious?

Reply:
Yes.
Motivation was that `check` command  based on exiting format was raising a restic error.
It seems 'restic check' does not parse parameters the same way as 'restic backup`.
To cope with this, I changed the syntax to be closer to restic documentation examples... and its more obvious, indeed.